packages/edit-post/src/components/secondary-sidebar/style.scss

```diff
@@ -38,10 +42,39 @@
 	height: $grid-unit-60;
 	padding-left: $grid-unit-20;
 	padding-right: $grid-unit-05;
+	ul {
+		width: calc(100% - #{ $grid-unit-50 });
+	}
+	li {
+		width: 50%;
+		& button {
+			width: 100%;
+		}
+	}
 }
 
 .edit-post-editor__list-view-panel-content {
-	overflow-y: auto;
 	// The table cells use an extra pixels of space left and right. We compensate for that here.
 	padding: $grid-unit-10 ($grid-unit-10 - $border-width - $border-width);
 }
+
+.edit-post-editor__list-view-overview {
+	position: absolute;
+	bottom: 0;
+	& > div > span:first-child {
+		width: 90px;
```

Comment:
Same as with 180px, a comment on what it is if we cannot male it a var of sorts.